### PR TITLE
docs(README): add requirements section

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +59,20 @@ Using [yarn](https://yarnpkg.com):
 yarn add @fingerprintjs/fingerprintjs-pro-react
 ```
 
+Using [pnpm](https://pnpm.js.org):
+
+```sh
+pnpm add @fingerprintjs/fingerprintjs-pro-react
+```
+
+## Requirements
```
Comment (JuroUhlar) [inline README.md:68]: Requirements go before "How to install" https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc

File: README.md [hunk 2]
```diff
@@ -58,6 +59,20 @@ Using [yarn](https://yarnpkg.com):
...
+
+- React 18 or higher
+- For Preact users: Preact 10.3 or higher
+- For Next.js users: Next.js 13.1 or higher
+- For Typescript users: Typescript 4.8 or higher
+
+
```
Comment (JuroUhlar) [inline README.md:75]: Include the FingerprintJS vs Pro section here: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#40df07e276694e3bba6c0c532690cdc2

File: README.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ Fingerprint Pro React SDK is an easy way to integrate **[Fingerprint Pro](https:
 ## Table of contents
 
 - [Installation](#installation)
+- [Requirements](#requirements)
```
Comment (JuroUhlar) [inline README.md:40]: Please align the intro with our template: https://www.notion.so/fingerprintjs/Writing-DX-GitHub-Readmes-WIP-9556ed587c0749b29f32a18017413cbc?pvs=4#ee9bf8b126bd4ccb8e91ff20b1fc70aa

**Review by JuroUhlar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheUnderScorer) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (TheUnderScorer) [reply README.md:76]: Done!

File: README.md (hunk 3 above)
Comment (TheUnderScorer) [reply README.md:38]: Done!

**Comment by TheUnderScorer** [comment]
@ilfa done!

File: README.md [hunk 4]
```diff
@@ -31,11 +31,11 @@
...
+Fingerprint is a device intelligence platform offering 99.5% accurate visitor identification. Fingerprint Pro React SDK is an easy way to integrate **[Fingerprint Pro](https://fingerprint.com/)** into your React application. It's also compatible with Next.js and Preact. See application demos in the [examples](https://github.com/fingerprintjs/fingerprintjs-pro-react/tree/main/examples) folder.  **This package works with Fingerprint Pro, it is not compatible with [open-source FingerprintJS](https://github.com/fingerprintjs/fingerprintjs).** You can learn more about the difference between Fingerprint Pro and open-source FingerprintJS in the [official documentation](https://dev.fingerprint.com/docs/pro-vs-open-source).
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply README.md]: Done!

File: README.md [hunk 5]
```diff
@@ -44,6 +44,15 @@ Fingerprint Pro React SDK is an easy way to integrate **[Fingerprint Pro](https:
 - [Support and feedback](#support-and-feedback)
 - [License](#license)
 
+## Requirements
+
+- React 18 or higher
+- For Preact users: Preact 10.3 or higher
```
[the remark this answers is not in the document]
Comment (TheUnderScorer) [reply README.md:50]: I'm not sure, @ilfa WDYT?

[the remark this answers is not in the document]
Comment (JuroUhlar) [reply README.md:50]: Got it, thanks!

**Review by JuroUhlar — APPROVED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**